tests/Client/TestRepo.php

```diff
@@ -57,7 +57,19 @@ public function fetchFile(string $fileName, int $maxBytes)
             }
             return $contents;
         } catch (\Exception $exception) {
-            return false;
+            throw new RepoFileNotFound("File $fileName not found.");
```

Comment:
The RepoFileNotFound exception should wrap the previous exception, to make debugging easier.

Reply:
fixed